lib/ansible/modules/include_tasks.py

```diff
@@ -18,9 +18,7 @@
 options:
   file:
     description:
-      - The name of the imported file is specified directly without any other option.
-      - Unlike M(ansible.builtin.import_tasks), most keywords, including loop, with_items, and conditionals, apply to this statement.
-      - The do until loop is not supported on M(ansible.builtin.include_tasks).
```

Comment:
These lines should probably be moved to `notes:` and not just deleted.

Reply:
Thanks for catching that. I think I just interpreted things the wrong way. I've added these lines back in and tried to tidy them up so they match the details in the import_tasks module docs.